week-4/Homework/mandatory/2-writers.js

```diff
@@ -39,6 +39,8 @@ let writers = [
   }
 ];
 
+message = writers.filter(x => x.alive).forEach(x => console.log(`Hi, my name is ${x.firstName} ${x.lastName}. I am ${x.age} years old, and work as a ${x.occupation}`))

```

Comment:
Well done!

Remember that [`forEach` does not return](https://codeyourfuture.github.io/syllabus-london/js-core-1/week-3/lesson.html#array-foreach) a new array. Therefore, your `message` variable will not contain anything useful. You don't need it - your code will work fine without it:

```javascript
writers.filter(w => w.alive)
       .forEach(w => console.log(`Hi, my name is...`))
```